mediaqueries-5/Overview.bs

```diff
@@ -478,6 +478,36 @@ Detecting the desire for less motion on the page: the 'prefers-reduced-motion' f
 
 	</dl>
 
+<h3 id="prefers-reduced-transparency">
+Detecting the desire to reduce the use of transparent or translucent layering effects on the page: the 'prefers-reduced-transparency' feature</h3>
```

Comment:
Could you shorten this title a bit? This a bit unwieldy, and does not need to reproduce the whole content of the section. How about:
> Detecting the desire for reduced transparency on the page: the 'prefers-reduced-transparency' feature